networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_mgmt.py

```diff
@@ -738,7 +738,7 @@ def qos_realize(self, qos, delete=False):
     def sg_members_realize(self, sg, delete=False):
         if delete and self.metadata(Provider.SG_RULES, sg.get("id")):
             LOG.warning(
-                "Resource:%s with ID:%s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
+                "Resource:%s with ID: %s deletion is rescheduled due to dependency.", Provider.SG_MEMBERS, sg.get("id")
```

Comment:
You didn't add the space after `Resource:` here.

Reply:
Added